core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.controller.ts

```diff
@@ -106,6 +106,8 @@ angular.module('oppia').controller('QuestionSuggestionEditorModalController', [
     // Checking if Question contains all requirements to enable
     // Save and Publish Question.
     $scope.isQuestionValid = function() {
+      $rootScope.$applyAsync();
+
```

Comment:
How many times is this invoked? Also, why do we need to force a digest loop here -- this method just checks if a question is valid, it does not look like it modifies any data?

Reply:
earlier this line is called infinite times, which is also causing e2e flakes, now this line is removed from this file code and placed to a more suitable file i.e. `question-editor.component.ts`.

Reply:
That's great, yes, we shouldn't be calling `applyAsync` more often than actually required..

Reply:
Done